edge/pkg/devicetwin/dtcontroller.go

```
@@ -135,7 +138,10 @@ func (dtc *DTController) distributeMsg(m interface{}) error {
 	if moduleName, exist := ActionModuleMap[message.Action]; exist {
 		//how to deal write channel error
 		log.LOGGER.Infof("Send msg to the %s module in twin", moduleName)
-		dtc.DTContexts.CommTo(moduleName, &message)
+		err := dtc.DTContexts.CommTo(moduleName, &message)
+		if err != nil {
+			return errors.New("can not communicate with module %v, error message is: %v", moduleName, err)
```

Do not wrap errors, please return the error directly.